[2023-03-13 22:20:05,421] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: test.process_user scheduled__2023-03-13T22:15:00+00:00 [queued]>
[2023-03-13 22:20:05,426] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: test.process_user scheduled__2023-03-13T22:15:00+00:00 [queued]>
[2023-03-13 22:20:05,426] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-03-13 22:20:05,427] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-03-13 22:20:05,427] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-03-13 22:20:05,434] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): process_user> on 2023-03-13 22:15:00+00:00
[2023-03-13 22:20:05,439] {standard_task_runner.py:52} INFO - Started process 7816 to run task
[2023-03-13 22:20:05,443] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'test', 'process_user', 'scheduled__2023-03-13T22:15:00+00:00', '--job-id', '151', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmp0u6jhc_m', '--error-file', '/tmp/tmpceec4y5p']
[2023-03-13 22:20:05,444] {standard_task_runner.py:80} INFO - Job 151: Subtask process_user
[2023-03-13 22:20:05,488] {task_command.py:369} INFO - Running <TaskInstance: test.process_user scheduled__2023-03-13T22:15:00+00:00 [running]> on host 5282528dd996
[2023-03-13 22:20:05,537] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=test
AIRFLOW_CTX_TASK_ID=process_user
AIRFLOW_CTX_EXECUTION_DATE=2023-03-13T22:15:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-13T22:15:00+00:00
[2023-03-13 22:20:05,553] {python.py:173} INFO - Done. Returned value was: None
[2023-03-13 22:20:05,564] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=test, task_id=process_user, execution_date=20230313T221500, start_date=20230313T222005, end_date=20230313T222005
[2023-03-13 22:20:05,620] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-03-13 22:20:05,646] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
